src/utils/data_types.h

```diff
@@ -14,7 +14,6 @@
 #include <cassert>
 
 #include "global_params.h"
-#include "openmvg_descriptor.hpp"
```

Comment:
Thanks a lot! This PR seems fine. But the primary issue here is that I currently mainly work on the 'feature' branch, which will be the next stable release. Since I just open-sourced this project not long ago, we have some coding style inconsistency between 'master' and 'feature'.(see https://github.com/hlzz/libvot/blob/feature/doc/coding_style.md for details) Also some problem in 'master' may have already been fixed in the 'feature' branch, such as the LIBVOT_USE_OPENCV issue. Anyway, guess I can first merge this PR and I will fix the inconsistency myself. Thanks!